tests/cucumber/steps/steps.js

```diff
@@ -180,6 +180,24 @@ module.exports = function getSteps(options) {
     return boxRefArray;
   }
 
+  let doRaw = false;
+
+  function doOrDoRaw(req) {
```

Comment:
We could do this if each step that made a call knew whether or not it should expect a valid response. To do that we either need to modify the step definition (doesn't make sense just to work for JS), or we can override the step like we do for some steps in this block https://github.com/algorand/js-algorand-sdk/blob/develop/tests/cucumber/steps/index.js#L436

However, since they're each a separate step definition we'd either need to use some string matching to tell, or override it separately for every API call. This global `doOrDoRaw` seemed the best option to me, but I'm open to suggestions on how to handle this better.

Reply:
I see now. I also took a look and don't have a better suggestion, but it wasn't immediately obvious why we had it, so a [comment like we have here](https://github.com/algorand/js-algorand-sdk/blob/bc272cb8048e9cf1ae3b311e2ac17da2ae35ff0f/tests/cucumber/steps/steps.js#L95) might be nice to explain the context of the global variable.